libgalois/include/katana/PropertyGraph.h

```diff
@@ -248,7 +309,11 @@ class KATANA_EXPORT PropertyGraph {
     }
   };
 
-  PropertyGraph();
+  PropertyGraph() = default;
+
+  PropertyGraph(
+      std::unique_ptr<tsuba::RDGFile> rdg_file, tsuba::RDG&& rdg,
+      std::unique_ptr<GraphTopology> topo_to_assign);
```

Comment:
Not for this PR: This should probably ALSO take a `&&` reference.

Reply:
Noted. Thanks.